ros2topic/ros2topic/verb/hz.py

```diff
@@ -82,6 +83,7 @@ def add_arguments(self, parser, cli_name):
             dest='use_wtime', default=False, action='store_true',
```

Comment:
TODO. using `Clock(clock_type=ClockType.STEADY_TIME)` can avoid the issue of time jumping.

Reply:
updated.

Reply:
I would recommend creating a new PR to address STEADY_TIME because it's not the requirement in this PR.

- The option without '-w' or '-s' is to use the clock with `STEADY_TIME`
- '-w' is to use the clock with `SYSTEM_TIME`(keep the behavior before)
- '-s' is to use `node.get_clock()`